llvm-bazel/.bazelrc

```diff
@@ -68,6 +68,23 @@ build:generic_gcc --copt=-Wno-misleading-indentation --host_copt=-Wno-misleading
 # Use `-Werror` for GCC to make sure warnings don't slip past.
 build:generic_gcc --copt=-Werror --host_copt=-Werror
 
+###############################################################################
+# Windows specific flags for building with GCC. Used on top of generic_gcc
+###############################################################################
+
+# Other windows toolchains have these by default
+build:mingw_gcc --cxxopt=-DHAVE_UNISTD_H=1 --host_cxxopt=-DHAVE_UNISTD_H=1
```

Comment:
I could run builds (at least partially) using clang-cl and msvc without those on `config.bzl` - it seems they're picked by default? 

Since only mingw needed them, I put them here instead. Wdyt? Or do you mean setting a bazel constraint set for when mingw is used?

(my compiler-foo-ness is subpar so bear with me if I'm missing something obvious :))

Reply:
> Or do you mean setting a bazel constraint set for when mingw is used?

I meant this. This is ok though.

What we should really do is detect the machine capabilities and encode it that way. The current method is really just a shortcut because Bazel is not well-equipped for that.